repo.go

```diff
@@ -22,22 +22,20 @@ type Interface interface {
 	// GetBundle returns the bundle referenced by curl.
 	GetBundle(curl *charm.URL) (charm.Bundle, error)
 
-	// Latest returns the latest revision of the charms referenced by curls,
-	// regardless of the revision set on each curl.
-	Latest(curls ...*charm.URL) ([]CharmRevision, error)
-
 	// Resolve resolves the series and revision of the given entity
 	// reference. If the series is not specified, it may be resolved
 	// by the charm store or rejected. After the series is resolved,
```

Comment:
This doesn't sound correct any more. We should clarify the behaviour here when the charm supports more than one series. Under what circumstances is it OK to return an empty list for the possible supported series, for example?